Code/Framework/AzFramework/AzFramework/Process/ProcessCommunicatorTracePrinter.cpp

```diff
@@ -80,7 +84,10 @@ void ProcessCommunicatorTracePrinter::WriteCurrentString(bool isFromStdErr)
         }
         else
         {
-            AZ_TracePrintf(m_window.c_str(), "%s", bufferToUse.c_str());
+            // Note that the input string will never have a newline at the end of it
+            // becuase it comes from ParseDataBuffer above, which splits by newline
```

Comment:
`becuase` -> `because`

Reply:
I always make this mistake.  I'll fix it :)